src/coredata.cs

```diff
@@ -733,25 +895,49 @@ public interface NSManagedObjectContext : NSCoding
 		void DetectConflictsForObject (NSManagedObject object1);
 
 		[Export ("observeValueForKeyPath:ofObject:change:context:")]
-		void ObserveValueForKeyPath (string keyPath, IntPtr object1, NSDictionary change, IntPtr context);
+#if XAMCORE_4_0
+		void ObserveValueForKeyPath (string keyPath, [NullAllowed] NSObject object1, [NullAllowed] NSDictionary<NSString, NSObject> change, IntPtr context);
```

Comment:
`ObserveValue`, double check but I'm fairly sure we used that name in other frameworks,